### DISCUSSION TITLE
stm32l562qeixq-pinctrl.dtsi file is missing

### DISCUSSION THREAD

**Discussion by dehaoji** [zephyrproject-rtos/zephyr#65104] [Q&A] [answered]
its included in the boards stm32I562e_dk file, but not listed under the dts/st/l5 folder. 

Can somebody please put it in?

Thanks,

**Comment by FRASTM** [comment, marked as answer]
It is located here, [stm32l562qeixq-pinctrl.dtsi ](https://github.com/zephyrproject-rtos/hal_stm32/blob/main/dts/st/l5/stm32l562qeixq-pinctrl.dtsi)

**Reply by dehaoji** [reply]
great, thanks.
